### DISCUSSION TITLE
Will this work with RDNA3?

### DISCUSSION THREAD

**Discussion by haveamission** [ChefKissInc/NootRX#97] [Q&A]
### macOS Version

Sequoia

### What is your CPU's model?

AMD Ryzen Z1 Extreme

### What is your GPU's model?

AMD 780M

### Please describe the feature in detail.

I would like NootRX to be able to support RDNA3 cards. I am happy to provide whatever help I can in this regard (dumps of various data, even possibly some code help if needed, though this is a bit closer to the metal than I typically write software)

### What alternatives have you tried, if any?

I am going to try disabling hardware acceleration, but that's obviously not a full workaround

**Comment by VisualEhrmanntraut** [comment]
RDNA 3 support would need a full reimplementation of the hardware abstraction code (HWLibs).

**Comment by menthorz** [comment]
How can a I view the gpu temp with NootRX.kext running?
